website/static/css/custom.css

```diff
@@ -26,6 +26,102 @@
     height: 80%;
 }
 
+/* Add special CSS for supported.md rendered on mobile */
+@media screen and (max-width: 480px) {
+    /* We have two styles for very narrow screens.
+    * We apply the hopscotch style to headless tables. It breaks
+    * tables down so that each cell acts like a <div>
+    *
+    * We apply a card style to tables with headings. This converts
+    * each cell into its own row, with a heading to its left.
+    */
+
+    .katex-cards table td {
+        display: block;
+        border: none;
+        border-bottom: 1px solid #eee;
+        position: relative;
+        padding-left: 50%;
+    }
+
+    .katex-hopscotch table td {
+        display: inline-block;
+        background: #fff;
+    }
+
+    .katex-hopscotch table tr {
+        border: none;
+        background: #fff;
+    }
+
+    .katex-cards table,
+    .katex-cards thead,
+    .katex-cards tbody,
+    .katex-cards th {
+        display: block;
+    }
+
+    /* Hide table headers (but not display: none;, for accessibility) */
+    .katex-cards thead tr {
+        display: block;
+        position: absolute;
+        top: -9999px;
+        left: -9999px;
+    }
```

Comment:
I confirmed that this is indeed best practice: https://webaim.org/techniques/css/invisiblecontent/#absolutepositioning

These (dis)appearing headers are really cool!